modules/fb_instant_articles_api/includes/admin.inc

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Settings for Facebook Instant Articles API module.
+ */
+
+/**
+ * Form constructor for Facebook Instant Articles API settings form.
+ */
+function fb_instant_articles_api_settings() {
+  $args = array(
+    '@development_mode' => 'https://developers.facebook.com/docs/instant-articles/api#create-fields',
+  );
+
+  // The Facebook app connection info could alternatively be stored via the fb
+  // module, but it's not yet ported to D8, and a bit cumbersome just for this.
+  $form['fb_instant_articles_api_app_id'] = array(
+    '#title' => t('Facebook App ID'),
+    '#type' => 'textfield',
+    '#default_value' => variable_get('fb_instant_articles_api_app_id'),
```

Comment:
@anbarasan1985  I suggest not, see my comment at line 16 for reasoning (also, the D7 fb module lets you add multiple apps… so we'd need to build more code to allow supporting that, which sadly I think would be more confusing than helpful. It's likely people will use a different fb app for instant articles than other fb integration).